### PR TITLE
feat: update the pt_BR translation

### PR DESCRIPTION
**Pull request by RedsonBr140**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [x] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [X] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

**Comment by Anuken** [comment]
Can someone review this? This translation has historically been rather controversial.

File: core/assets/bundles/bundle_pt_BR.properties [hunk 1]
```diff
@@ -183,7 +183,8 @@ filename = Nome do arquivo:
 unlocked = Novo bloco desbloqueado!
 available = Nova pesquisa disponível!
 unlock.incampaign = < Desbloqueie na campanha para mais detalhes >
-campaign.select = Select Starting Campaign
+campaign.select = Selecione a campanha inicial
+
```
Comment (DinoWattz) [inline core/assets/bundles/bundle_pt_BR.properties:187]: There's an extra added line here

File: core/assets/bundles/bundle_pt_BR.properties [hunk 2]
```diff
@@ -1785,47 +1786,49 @@ hint.presetDifficulty = Esse setor tem um [scarlet]alto nível de ameaça inimig
...
+onset.research2 = Comece a pesquisar [accent]fábricas[].\nPesquise o \uf74d [accent]Esmagador de Penhasco[] e o \uf779 [accent]silicon arc furnace[].
+onset.arcfurnace = O arc furnace precisa de \uf834 [accent]areia[] e \uf835 [accent]grafite[] para criar \uf82f [accent]silício[].\n[accent]Energia[] também é necessária.
+onset.crusher = Use o \uf74d [accent]Esmagador de Areia[] para minerar areia.
+onset.fabricator = Use [accent]unidades[] para explorar o mapa, defender construções e atacar o inimigo. Pesquise e coloque um \uf6a2 [accent]Fabricador de Tanques[].
+onset.makeunit = Produza uma unidade.\nUse o botão "?" para ver os requisitos da fábrica selecionada.
+onset.turrets = Unidades são efetivas, mas [accent]torretas[] proveem melhores capacidades defensivas se usadas efetivamente.\nColoque uma torreta \uf6eb [accent]Breach[].\nTorretas requerem \uf748 [accent]munição[].
+onset.turretammo = Abasteça a torreta com [accent]munição de berílio.[]
+onset.walls = [accent]Muros[] podem previnir danos recebidos de atingir as construções.\nColoque \uf6ee [accent]muros de berílio[] em volta das torretas.
+
```
Comment (DinoWattz) [inline core/assets/bundles/bundle_pt_BR.properties:1826]: Extra line

File: core/assets/bundles/bundle_pt_BR.properties [hunk 3]
```diff
@@ -1785,47 +1786,49 @@ hint.presetDifficulty = Esse setor tem um [scarlet]alto nível de ameaça inimig
...
+onset.enemies = Inimigo vindo, se prepare.
+onset.attack = O inimigo está vulnerável. Contra ataque.
+onset.cores = Novos núcleos podem ser colocados em [accent]ladrilhos de núcleo[].\nNovos núcleos funcionam como bases avançadas e compartilham seus recursos com outros núcleos.\nColoque um \uf725 núcleo.
+onset.detect = O inimigo poderá te detectar em 2 minutos.\nConstrua defesas, mineração e produção.
+
```
Comment (DinoWattz) [inline core/assets/bundles/bundle_pt_BR.properties:1831]: Extra line

**Review by DinoWattz — CHANGES_REQUESTED** [review]
The translation looks right but there are extra lines added for no reason

File: core/assets/bundles/bundle_pt_BR.properties (hunk 2 above)
Comment (DinoWattz) [inline core/assets/bundles/bundle_pt_BR.properties:1826]:
```suggestion
```

File: core/assets/bundles/bundle_pt_BR.properties (hunk 3 above)
Comment (DinoWattz) [inline core/assets/bundles/bundle_pt_BR.properties:1831]:
```suggestion
```

**Pull request merged**